zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -223,9 +223,6 @@ api.options.addr.dialog.remove.text = Are you sure you want to remove the select
 api.options.addr.dialog.remove.button.cancel = Cancel
 api.options.addr.dialog.remove.button.confirm = Remove
 api.options.addr.dialog.remove.checkbox.label = Do not show this message again
-api.options.addr.table.header.enabled	= Enabled
-api.options.addr.table.header.regex	= Regex
-api.options.addr.table.header.value	= Address
```

Comment:
Why were these API keys/values removed?

Reply:
Those keys/values are duplicated..
[line 219](https://github.com/zaproxy/zaproxy/blob/cfb89797be118cc9231e81db0c7dc090aec5eafe/zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties#L219) == [line 226](https://github.com/zaproxy/zaproxy/blob/cfb89797be118cc9231e81db0c7dc090aec5eafe/zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties#L226)
[line 220](https://github.com/zaproxy/zaproxy/blob/cfb89797be118cc9231e81db0c7dc090aec5eafe/zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties#L220) == [line 227](https://github.com/zaproxy/zaproxy/blob/cfb89797be118cc9231e81db0c7dc090aec5eafe/zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties#L227)
[line 221](https://github.com/zaproxy/zaproxy/blob/cfb89797be118cc9231e81db0c7dc090aec5eafe/zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties#L221) == [line 228](https://github.com/zaproxy/zaproxy/blob/cfb89797be118cc9231e81db0c7dc090aec5eafe/zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties#L228)

Reply:
I'd suggest doing these changes in other PR for clarity.